e2e/tests/router-guards.spec.ts

```
@@ -0,0 +1,72 @@
+    await page.goto("/chat/<script>alert(1)</script>");
+    // Guard rejects the malformed ID and redirects to /chat
+    await page.waitForURL(/\/chat/);
+    // The XSS payload should not be in the URL
+    expect(page.url()).not.toContain("<script>");
+    // The page should render normally (no crash)
+    await expect(page.locator('[data-testid="app-title"]')).toBeVisible();
+  });
+
+  test("path traversal in sessionId → redirects to /chat", async ({
+    page,
+  }) => {
```

ESLint

[error] Replace `⏎····page,⏎·` with `·page`

(prettier/prettier)